doc/src/build/cli-client.md

```diff
@@ -32,15 +32,15 @@ The Sui Client CLI supports the following commands:
 | `pay_all_sui`        | Pay all residual SUI coins to the recipient with input coins, after deducting the gas cost. The input coins also include the coin for gas payment, so no extra gas coin is required.                                                          |
 | `pay_sui`            | Pay SUI coins to recipients following specified amounts, with input coins. Length of recipients must be the same as that of amounts. The input coins also include the coin for gas payment, so no extra gas coin is required.                 |
 | `publish`            | Publish Move modules.                                                                                                                                                                                                                         |
+| `replay-transaction` | Replay a given transaction to view transaction effects. Set environment variable MOVE_VM_STEP=1 to debug.                                                                                                                                     |
```

Comment:
The new docs are also updated by Ronny, but it'd be great to add an example too. Can be done in a follow up PR: https://github.com/MystenLabs/sui/blob/main/docs/content/references/cli/client.mdx